src/routes/files.ts

```diff
@@ -23,11 +23,11 @@ export default async function files(fastify: FastifyInstance) {
   fastify.get('/files/credentials', async (req, reply) => {
     const sessionCache: SessionCache = req.sessionCache;
     const results = new Map<ContactType, String[][]>();
+    const hierarchyProps = new Map<ContactType, string[]>();
     const places = sessionCache.getPlaces();
     places.forEach(place => {
-      const parent = Config.getParentProperty(place.type);
       const record = [
-        place.hierarchyProperties[parent.property_name],
+        ...Object.values(place.hierarchyProperties),
```

Comment:
nit: if there are no `replacements` in the resulting data, should we include it?

Reply:
That'd complicate the code quite a bit. it'd only take one replacement to then have to start checking if we should be adding an empty column to all the other records and headers

Reply:
cool. no worries